INSTALL.pipeline

```diff
@@ -6,19 +6,29 @@ To run the pipeline, you will need:
 - seesaw (automatically installed by Pip)
 - rsync
 - wpull (automatically installed by Pip)
-- PhantomJS 2.1.1
+- youtube-dl
+- PhantomJS 2.1.1 (or 1.9.8 if that doesn't work for you)
 
-Quick install, for Debian and Debian-esque systems:
+Quick install, for Debian and Debian-esque systems like Ubuntu:
 
     sudo apt-get update
     sudo apt-get install build-essential python3-dev python3-pip \
       libxml2-dev libxslt-dev zlib1g-dev libssl-dev libsqlite3-dev \
       libffi-dev git tmux fontconfig-config fonts-dejavu-core \
-      libfontconfig1 libjpeg-turbo8 libjpeg8 lsof
+      libfontconfig1 libjpeg-turbo8 libjpeg8 lsof ffmpeg youtube-dl
```

Comment:
Indeed it is.  From the youtube-dl docs:
"Since the end of April 2015 and version 2015.04.26, youtube-dl uses -f bestvideo+bestaudio/best as the default format selection (see #5447, #5456). If ffmpeg or avconv are installed this results in downloading bestvideo and bestaudio separately and muxing them together into a single file giving the best overall quality available."